modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -21,6 +21,11 @@ endif::postinstall-bare-metal[]
 ifdef::postinstall-bare-metal[]
 As an alternative to using the `configure-ovs.sh` shell script to set a `br-ex` bridge on a bare-metal platform, you can create a `NodeNetworkConfigurationPolicy` (NNCP) custom resource (CR) that includes an NMState configuration file. 
 
+[NOTE]
+====
+As a postinstallation task, do not make configuration changes to the `br-ex` bridge or its underlying interfaces.
```

Comment:
I don't think we want this note here. This is the section of the docs where we discuss the changes you _can_ make when you deploy with a custom NMState br-ex.